release notes/v0.27.0.md

```diff
@@ -0,0 +1,81 @@
+k6 v0.27.0 is here! :tada:
+
+This is a milestone release containing a major overhaul to the execution subsystem of k6, along with many improvements and bug fixes.
+
+## New features and enhancements!
+
+### New execution engine (#1007)
+
+After 1.5 years in the making, the k6 team is proud to release the first public version of the new execution engine, offering users new ways of modeling advanced web testing scenarios that can more closely represent real-world traffic patterns.
+
+These new scenarios are entirely optional, and existing k6 scripts and options should continue to work the same as before (with a few minor breaking changes, see below), but please [create a bug issue](https://github.com/loadimpact/k6/issues/new?labels=bug&template=bug_report.md) if that's not the case.
+
+See the [documentation](https://k6.io/docs/using-k6/executors) for details and examples, or keep reading for the summary.
+
+#### Execution scenarios
+
+Some of the currently possible scenarios were formalized into standalone executors:
+
+- Shared iterations: a fixed number of iterations are "shared" by all VUs, and the test ends once all iterations are executed. This executor is equivalent to the global `vus` and `iterations` options.
+- Constant VUs: a fixed number of VUs execute as many iterations as possible for a specified amount of time. This executor is equivalent to the global `vus` and `duration` options.
+- Ramping VUs: a variable number of VUs execute as many iterations as possible for a specified amount of time. This executor is equivalent to the global `stages` option.
+- Externally controlled: control and scale execution at runtime via k6's REST API or the CLI.
+
+And new executors were added to support some of the most frequently requested scenarios that were previously difficult or impossible to model:
+
+- Per VU iterations: each VU executes a fixed number of iterations (#381).
+- Constant arrival rate: a fixed number of iterations are executed in a specified period of time. This allows k6 to dynamically change the amount of VUs during a test run to achieve the specified amount of iterations per period, which is useful for a more accurate representation of RPS, for example. See #550 for details.
+- Ramping arrival rate: a variable number of iterations are executed in a specified period of time. This is similar to the ramping VUs executor, but for iterations instead.
+
```

Comment:
```suggestion

It's important to also note that all of these executors, except the `externally-controlled` one, can be used both in local k6 execution with `k6 run`, and in the distributed cloud execution with `k6 cloud`. This even includes "old" executors that were previously unavailable in the cloud, like the `shared-iterations` one. Now, you can execute something like `k6 cloud --iterations 10000 --vus 100 script.js` without any issues.

```

Reply:
Maybe additionally add that there is a default maxDuration of 10min in this case?

Reply:
hm Iwould also consider this to be a breaking change as well - if you previously calld `k6 run -i 10000 -u 5 script.js` and it took 10h that would be fine, but now it will get stopped after 10minutes (and 30 seconds)